.github/workflows/ci.yml

```diff
@@ -0,0 +1,324 @@
+      - name: Cache node modules
+        uses: actions/cache@v1
+        with:
+          path: /lektor/admin/node_modules
+          key: ${{ runner.os }}-node-${{ matrix.NODEJS_VERSION }}-modules-${{ hashFiles('**/lektor/admin/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-${{ matrix.NODEJS_VERSION }}-modules-
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.NODEJS_VERSION }}
+      - name: Install node dependencies
+        run: |
+          cd lektor/admin/
+          npm install .
+          npm run webpack
+      - name: Show node environment
+        run: |
+          node --version
+          npm --version
+
+  mac-node:
+    name: Mac node${{ matrix.NODEJS_VERSION }} tests
+    runs-on: macos-latest
+    needs: lint-js
+    env:
+      CI: True
+      NODEJS_VERSION: ${{ matrix.NODEJS_VERSION }}
+    strategy:
+      fail-fast: false 
+      matrix:
+        NODEJS_VERSION: ['10', '12', '13']
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+      - name: Cache node
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ matrix.NODEJS_VERSION }}-${{ hashFiles('**/lektor/admin/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-${{ matrix.NODEJS_VERSION }}-
+      - name: Cache node modules
+        uses: actions/cache@v1
+        with:
+          path: /lektor/admin/node_modules
+          key: ${{ runner.os }}-node-${{ matrix.NODEJS_VERSION }}-modules-${{ hashFiles('**/lektor/admin/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-${{ matrix.NODEJS_VERSION }}-modules-
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.NODEJS_VERSION }}
+      - name: Install node dependencies
+        run: |
+          cd lektor/admin/
+          npm install .
+          npm run webpack
+      - name: Show node environment
+        run: |
+          node --version
+          npm --version
+
+  win-node:
+    name: Win node${{ matrix.NODEJS_VERSION }} tests
+    runs-on: windows-latest
+    needs: lint-js
+    env:
+      CI: True
+      NODEJS_VERSION: ${{ matrix.NODEJS_VERSION }}
+    strategy:
+      fail-fast: false 
+      matrix:
+        NODEJS_VERSION: ['10', '12', '13']
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+      - name: Cache node
+        uses: actions/cache@v1
+        with:
+          path: ~\AppData\Roaming\npm-cache
+          key: ${{ runner.os }}-node-${{ matrix.NODEJS_VERSION }}-${{ hashFiles('**\lektor\admin\package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-${{ matrix.NODEJS_VERSION }}-
+      - name: Cache node modules
+        uses: actions/cache@v1
+        with:
+          path: /lektor/admin/node_modules
```

Comment:
Will forwardslashes really work here?

Reply:
I will leave the node_modules cache for a separate PR as well. I am not really sure about it